client/signup/config/flows.js

```diff
@@ -266,21 +258,8 @@ function filterDesignTypeInFlow( flow ) {
 function filterFlowName( flowName ) {
 	const defaultFlows = [ 'main', 'website' ];
 
-	/**
-	 * Only run the User First Signup for logged out users.
-	 */
-	if ( ! user.get() ) {
-		if ( includes( defaultFlows, flowName ) && abtest( 'userFirstSignup' ) === 'userFirst' ) {
-			return 'userfirst';
-		}
-
-		/**
-		 * Users should not be able to reach `userfirst-secondary` without being logged in, since
-		 * it doesn't contain the `user` step to register a user and site respectively.
- 		 */
-		if ( flowName === 'userfirst-secondary' ) {
-			return 'userfirst';
-		}
+	if ( includes( defaultFlows, flowName ) && abtest( 'userFirstSignup' ) === 'userFirst' ) {
```

Comment:
We need to limit the test to logged out users. 

With the current way, it will include users who try to create a new site while logged in and it may skew the results.

The logic that was set before with: `if ( ! user.get() ) { ... }` can be reused here.